classes/class-excludes.php

```diff
@@ -195,10 +195,6 @@ function( $exclude ) {
 	 */
 	public function is_file_excluded( \SplFileInfo $file ) {
 
-		if ( ! $file->isFile() ) {
```

Comment:
@willmot What I got confused about is that `$files` array is getting values from `list_directory_by_total_filesize()` https://github.com/humanmade/backupwordpress/blob/master/admin/schedule-form-excludes.php#L114 which internally uses `Site_Size->filesize( \SplFileInfo $file )` and that takes `\SplFileInfo` param, which according to docs is `...offers a high-level object oriented interface to information for an individual file`.

That's why I got confused as to why does it work for directories? Shall I still rename my newly added function to `is_path_excluded`?